src/components/Nuclie/ProductCard.js

```diff
@@ -122,6 +122,9 @@ const ProductCard = (props) => {
             <p className='crnt-price'>₹{price}</p>
             <p className='prev-price'>₹{prevPrice}</p>
             <p className='discount'>{discount}% off</p>
+            <p className='rating'>
+              {product.rating} <img src={Star} alt='rating_star' /> 5
```

Comment:
Can destructure rating directly since you are already destructuring other properties.

Reply:
Also, you dont have to show 5 here. Just show the actual rating. 
4 star 5 does not make sense on the product card.